lookout/style/format/analyzer.py

```diff
@@ -27,6 +25,9 @@
 from lookout.style.format.rules import Rules, TrainableRules
 from lookout.style.format.utils import generate_comment, merge_dicts
 from lookout.style.format.virtual_node import VirtualNode
+import numpy
```

Comment:
Wow this is a bug in PyCharm import ordering

Reply:
I'll fix it. (import order, not the bug in PyCharm :) )